docs/content/getting-started/_index.md

```diff
@@ -63,7 +63,7 @@ Different components have different intended uses. We tend to think of them in 3
 The beauty of web components is that they have much of the styling built-into the tag itself. Start with the tags first.
 
 ```
-<body>
+<body unresolved>
```

Comment:
I would prefer that the body tag be removed from this example.  I don't think it adds anything, since the object of the example is pfe-cta.  Unless I'm missing something about why `<body>` is relevant in this example, I think this looks better:

```html
    <pfe-cta>
        <a href="#">Learn more</a>
    </pfe-cta>
```

Reply:
agreed; pushed in 05f2510de4c46a5587cf9f695481e22269f966da